src/components/Form/Input.tsx

```diff
@@ -21,6 +22,7 @@ export const Input: FC<FormInputFieldProps> = ({
       label={label}
       helpText={helpText}
       error={error}
+      className={groupClassName}
```

Comment:
I see the benefit, but I'm a bit uncomfortable about this API. What if other sub elements need the `className` set, or some other prop? In this case it's not a big deal since there is only a `Group` and `Input`, but just thinking about other components, and keeping a consistent API.

Open to other ideas as well :)